{{cookiecutter.project_slug}}/.ebextensions/django.config

```diff
@@ -70,4 +70,11 @@ files:
       "\e[A":history-search-backward
       ## arrow down
       "\e[B":history-search-forward
+
+  "/opt/elasticbeanstalk/tasks/taillogs.d/cfn-init-cmd.conf":
+    mode: "000755"
```

Comment:
:question: Does it need to be 755, or can it be 644?

Reply:
@zags Does this need to be resolved? Not sure if you saw this before merging.

Reply:
EB docs say 755 https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/using-features.logging.html